purchase_work_acceptance/views/account_move_views.xml

```diff
@@ -3,7 +3,7 @@
     <record id="view_move_form" model="ir.ui.view">
         <field name="name">account.move.form</field>
         <field name="model">account.move</field>
-        <field name="inherit_id" ref="account.view_move_form" />
+        <field name="inherit_id" ref="purchase_stock.stock_move_purchase" />
```

Comment:
maybe it would be better to use `purchase.view_move_form_inherit_purchase`

Reply:
You are right! Thanks!